.github/config.yml

```diff
@@ -208,14 +208,7 @@ run:
     - tools/code_format/check_format.py
   mobile-cc:
     paths:
-    - .bazelrc
-    - .bazelversion
-    - .github/config.yml
-    - bazel/external/quiche.BUILD
-    - bazel/repository_locations.bzl
-    - mobile/.bazelrc
-    - mobile/**/*
-    - tools/code_format/check_format.py
+    - "**/*"
```

Comment:
as discussed offline can we make this a bit more specific - something like

```yaml

    - .bazelrc
    - .bazelversion
    - .github/config.yml
    - bazel/external/quiche.BUILD
    - bazel/repository_locations.bzl
    - mobile/**/*
    - source/**/*
    - test/?/*
    - envoy/**/*
    - api/**/*
```

probably we should update the others that run on everything similarly

Reply:
thanks for the suggestion, made the change, let's see how CI does with it